packages/flutter/test/material/search_test.dart

```diff
@@ -137,6 +137,47 @@ void main() {
     expect(find.text('Suggestions'), findsOneWidget);
   });
 
+  testWidgets('Can close search with escape button and return null', (WidgetTester tester) async {
+    final _TestSearchDelegate delegate = _TestSearchDelegate();
+    addTearDown(() => delegate.dispose());
+    final List<String?> selectedResults = <String?>[];
+
+    await tester.pumpWidget(TestHomePage(delegate: delegate, results: selectedResults));
+
+    // We are on the homepage
```

Comment:
nit: end comments with a period, here and other comments in this test.

Reply:
Hey @Renzo-Olivares! I used the test `Can close search with system back button to return null` as base. Should I add the period in the others too?

Reply:
That would be great, but just having it in this test would be okay too.

Reply:
@Renzo-Olivares I updated the comments.